main.js

```diff
@@ -515,6 +509,8 @@ class Covid19 extends utils.Adapter {
 						await this.localDeleteState(`Germany._Impfungen.rkiImpfungePflegeheim`);
 						await this.localDeleteState(`Germany._Impfungen.rkiImpfungenPflegeheim`);
 
+					} else {
+						this.log.warn(`Cannot handle vaccination data for Germany fromm RKI, if this erros continues please report a bug to the developen!`);
```

Comment:
diesen log Eintrag hatte ich in 0.7.1 optimalisiert

Reply:
passt. ich muss den neuen master nochmal mergen.

Reply:
:) dan warte ich darauf.... sonst kan es so übernommen werden nur schade das auch hier die Bundesländer nicht dabei sind.
Und man könnte die Impfdaten so fuer alle Ländern übernehmen :)

Reply:
@DutchmanNL der neue master ist drin

Reply:
@DutchmanNL genau. alle anderen länder gehen im nächsten schritt auch. aber man ist sich ja erstmal selbst der nächste :-D
ich finde die impfdaten für bundesländer einfach komplett unspannend. die sind ja eh sehr ähnlich zu gesamt deutschland...